config/unicorn.conf.rb

```diff
@@ -25,7 +25,7 @@
 # feel free to point this anywhere accessible on the filesystem
 pid (ENV["UNICORN_PID_PATH"] || "#{discourse_path}/tmp/pids/unicorn.pid")
 
-if ENV["RAILS_ENV"] == "development" || !ENV["RAILS_ENV"]
+if ENV["RAILS_ENV"] == "test" || !ENV["RAILS_ENV"]
```

Comment:
Maybe we use `not production`?

Reply:
This was just for the debugging purposes, but I might leave it in, so I will change it 🙂 👍